### PR TITLE
Updated calls to avoid log spamming

### PR DESCRIPTION
**Pull request by japokorn**
Updated deprecated get_info calls that caused
excessive logging of warnings

### CODE REVIEW

**Comment by M4rtinK** [comment]
Thanks a lot - this should hopefully make the Anaconda logs (more?) readable again. :)

**Comment by vpodzime** [comment]
> Thanks a lot - this should hopefully make the Anaconda logs (more?) readable again. :)

LOL :)

File: blivet/populator/populator.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def _udev_device_is_disk(self, info):
             udev/sysfs, we have to define what is a disk in terms of what is
             not a disk.
         """
-        return udev.device_is_realdisk(info)
+        return udev.device_is_disk(info)
```
Comment (vpodzime) [inline blivet/populator/populator.py]: This looks quite suspicious, please double-check it's no change in the behavior.

Comment (japokorn) [inline blivet/populator/populator.py]: You are right. Deprecation warning popping out is targeted to version 3.0. This part of code is to be left unchanged.

**Comment by japokorn** [comment]
Jenkins, test this please.

**Comment by japokorn** [comment]
Jenkins, test this please.

**Comment by vpodzime** [comment]
Jenkins, test this, please.

**Comment by vpodzime** [comment]
Is this really done? I mean, are these the only places in the codebase where we use the deprecated API?

**Comment by M4rtinK** [comment]
I guess I'll check with the branch in updates image. :)

**Comment by M4rtinK** [comment]
So I've pulled the 2.1-devel-udev_log_spam branch to an updates image and it does seem like most of the udev warning spam is still there (log from a manual installation):
https://paste.fedoraproject.org/550675/86492662/

I *think* some of the warnings that did show up before are not there anymore, but the overall amount still has not really decreased & the log is still not very readable.

**Comment by dwlehman** [comment]
My hunch is this will require a change everywhere we access `info` or `data` as a dict, whether by key lookup (eg: `info['ID_FS_TYPE']`) or by `dict.get` (eg: `info.get('ID_FS_TYPE', '')`). Most of them will be in `blivet/udev.py` but there will certainly be others sprinkled throughout `blivet/populator/`.

**Comment by vpodzime** [comment]
> My hunch is this will require a change everywhere we access info or data as a dict, whether by key lookup (eg: info['ID_FS_TYPE']) or by dict.get (eg: info.get('ID_FS_TYPE', '')). Most of them will be in blivet/udev.py but there will certainly be others sprinkled throughout blivet/populator/.

Let me repeat my suggestion once more here -- why don't we return the dicts (``properties``) instead of original objects right when we iterate over them? We don't need the objects themselves anywhere, do we?

**Pull request closed without merging**